test/Microsoft.SqlTools.ServiceLayer.Test.Common/ConnectionSetting.cs

```diff
@@ -58,6 +58,12 @@ public InstanceInfo(string versionKey)
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public string ProfileName { get; set; }
 
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string Encrypt { get; set; }
```

Comment:
Why not use the boolean encrypt and strictEncryption options like you added above? Not a huge deal so if you'd prefer to keep this closer to the SqlClient implementation then that's fine - I just usually find it easier to keep things as consistent as possible within a code base.

Reply:
I wanted to leave the surface API as close to how it will be captured by connection strings by ADS end users and MDS as well. The bool props are mainly intended to keep the internal models backwards compatible without surfacing MDS new encryption object.